eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/acl/Acl.java

```diff
@@ -26,14 +26,25 @@
 import org.apache.commons.lang3.StringUtils;
 
 import java.util.Properties;
-import java.util.ServiceLoader;
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class Acl {
+
     private static final Logger logger = LoggerFactory.getLogger(Acl.class);
-    private static AclService aclService;
+
+    private static final Acl INSTANCE = new Acl();
+
+    private AclService aclService;
+
+    private Acl() {
+
+    }
+
+    public static Acl getInstance() {
```

Comment:
Suggest merge getInstance() and init() method.
eg. public static Acl getInstance(String aclPluginType) .
Add collection for Acl instannce and pluginType mapping.

Reply:
@jonyangx  Thanks for you suggest. I will do it and resubmit later!

Reply:
@mxsm Pls Consider start() and stop()  methods, how to manage thd aclService lifecycle.

Reply:
@jonyangx  OK, I will